bims/api_views/date_category_filter.py

```diff
@@ -0,0 +1,17 @@
+# coding=utf-8
+import json
+from django.http.response import HttpResponse
+from rest_framework.views import APIView
+from bims.models.biological_collection_record import BiologicalCollectionRecord
+
+
+class DateCategoryList(APIView):
```

Comment:
i see this class is for returning category. Maybe remove "Date" on the name of class